www/i18n/en.json

```diff
@@ -362,6 +359,8 @@
         "survey-missing-formpath": "Error while fetching resources in config: survey_info.surveys has a survey without a formPath"
     },
     "errors": {
+      "registration-check-token": "User registration error. Please check your token and try again.",
+      "unable-preload-not-registered": "Unable to preload survey before registration",
```

Comment:
Agreed, that should never happen. This is just a safeguard to tell the user something is wrong, in case it does happen.

I meant "Unable to X" as in "X is not a thing that should happen", but I can see how it's being interpreted as "We tried to do X and it was unsuccessful"

Reply:
reworded